tls/s2n_connection.c

```diff
@@ -934,6 +955,20 @@ int s2n_connection_set_write_fd(struct s2n_connection *conn, int wfd)
     return 0;
 }
 
+int s2n_connection_get_write_fd(struct s2n_connection *conn, int *writefd)
+{
+    POSIX_ENSURE_REF(conn);
+    POSIX_ENSURE_REF(writefd);
+
+    /* in case we do not use managed io, or in case write fd was never set we should return invalid fd */
+    if(conn->managed_io && conn->send_io_context) {
```

Comment:
Nit:
```suggestion
    /* in case we do not use managed io, or in case write fd was never set we should return invalid fd */
    if (conn->managed_io && conn->send_io_context) {
```

(Also-- is this comment useful?)

Reply:
fixed and removed the comment